addons/l10n_tw_edi_ecpay/models/res_partner.py

```diff
@@ -7,3 +7,12 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     invoice_edi_format = fields.Selection(selection_add=[('tw_ecpay', "ECPay")])
+
+    def _compute_is_company(self):
+        l10n_tw_partners = self.filtered(lambda p: p.vat and p.country_code == 'TW')
```

Comment:
As for other overrides, would take TW partners, then below check vat is set otherwise company is automatically False. Allows to contain TW partners (by the way, we agree TW is TomorroWland ?).